bin/pycbc_live

```diff
@@ -359,6 +361,55 @@ class LiveEventManager(object):
                 cmd, shell=True, stdout=logfile, stderr=logfile
             )
 
+    def create_gdb(self):
+        """
+        Function to create GraceDB session.
+        """
+        from ligo.gracedb.rest import GraceDb
+        logging.info('Creating GraceDB session')
+        # Set up dict for args that reload the certificate if it will expire in the reload_buffer time
+        # These args are documented here: https://ligo-gracedb.readthedocs.io/en/latest/api.html#ligo.gracedb.rest.GraceDb
+        # Because we do not change any of the request session values when running the code, it should remain thread safe. 
+        gdbargs = {'reload_certificate': True, 'reload_buffer': 300}
```

Comment:
I am not sure what the impact of reloading the cert is (probably very small) but 300 s seems a bit too often. I propose an hourly reload.

Reply:
Ah, the `reload_buffer` function works slightly differently to that. When it's set to 300s it means that whenever the GraceDb client interacts with the server, it checks whether the certificate expiration time is within the next 300s (or if you've passed it), and if so, reloads the certificate from disk before starting your server interaction. I haven't been able to test that the cert reloads yet, but I can do a test asap?

I'm thinking of doing this by:
1) Creating a new cert with a short lifespan.
2) Starting python, creating the connection object, and connecting to GraceDb with it.
3) Waiting until the cert expires, then recreating it on disk.
4) Trying to connect to GraceDb again without recreating the connection object.

I'm pretty sure they have done a very similar test in the development of the GraceDb Client package to ensure the code works :-)

Reply:
Ok. Then it seems we can test by simply letting the analysis run for one week, and making sure no uploads fail.